configs/default/HGLR-HGLRCF405_V2.config

```diff
@@ -0,0 +1,124 @@
+# Betaflight / STM32F405 (S405) 4.4.2 May 31 2023 / 22:53:28 (23d066d) MSP API: 1.45
+
+#define USE_GYRO
+#define USE_GYRO_SPI_MPU6000
+#define USE_ACC
+#define USE_ACC_SPI_MPU6000
+#define USE_BARO
+#define USE_BARO_BMP280
+#define USE_BARO_DPS310
+#define USE_MAX7456
+#define USE_FLASH
+#define USE_FLASH_W25Q128FV
+
+board_name HGLRCF405_V2
```

Comment:
Please remove this _ bar like we talked before

Reply:
OK